api/proto/info/info.proto

```diff
@@ -1024,3 +1026,33 @@ message ZInfoEdgeview {
   // total number of app related cmds edge-view has processed
   uint32 count_app = 4;
 }
+
+// LocReliability - reliability of location information.
+enum LocReliability {
+  LOC_RELIABILITY_UNSPECIFIED = 0;
+  LOC_RELIABILITY_VERY_LOW = 1;
+  LOC_RELIABILITY_LOW = 2;
+  LOC_RELIABILITY_MEDIUM = 3;
+  LOC_RELIABILITY_HIGH = 4;
+}
+
+// ZInfoLocation - geographic location of the device obtained from a GNSS receiver.
+message ZInfoLocation {
+  // Latitude in the Decimal degrees (DD) notation.
+  double latitude = 1;
+  // Longitude in the Decimal degrees (DD) notation.
+  double longitude = 2;
+  // Altitude w.r.t. mean sea level in meters.
+  double altitude = 3;
+  // UTC timestamp for the location measurement
+  // (recorded by the GNSS clock).
+  google.protobuf.Timestamp utc_timestamp = 4;
```

Comment:
Is this in the maximum clock resolution from the GNSS? You had some comment in the document about milliseconds but I assume we can get higher resolution.

Reply:
The resolution of this timestamp is down to milliseconds. This is the best that the QMI protocol provides: https://github.com/linux-mobile-broadband/libqmi/blob/main/data/qmi-service-loc.json#L268-L276
The receiver can do probably better and for the purpose of time synchronization we would have to find different means than QMI.

Reply:
How about documenting the fact that this is millisecond resolution only (due to QMI) in the protobuf and in the markdown file(s)?

Reply:
Done - added comment to both proto and MD file